test/phoenix_ecto/html_test.exs

```diff
@@ -204,10 +254,6 @@ defmodule PhoenixEcto.HTMLTest do
       assert input_value(f, :integer) == 123
       assert input_value(f, :string) == "string"
       assert input_value(f, :float) == 78.9
-
-      assert input_value(f, :integer, 0) == 123
-      assert input_value(f, :string, "default") == "default"
-      assert input_value(f, :float, 0.0) == 78.9
```

Comment:
This has been removed because we don't have `input_value/3` on Phoenix HTML anymore.